phpstan.neon.dist

```diff
@@ -0,0 +1,17 @@
+includes:
+    - ./vendor/nunomaduro/larastan/extension.neon
+    - ./phpstan-baseline.neon
+
+parameters:
+    tmpDir: .cache/phpstan
```

Comment:
Will speedup subsequent runs, in a Github Workflow we'd need to use the cache action to store this cache